io-engine/src/subsys/nvmf/subsystem.rs

```diff
@@ -165,6 +196,186 @@ fn make_sn<T: AsRef<[u8]>>(uuid: T) -> CString {
 }
 
 impl NvmfSubsystem {
+    pub fn extract_nexus_name(nqn: &str) -> Option<String> {
+        let vec: Vec<String> = nqn.split(':').map(|s| s.to_string()).collect();
+
+        if vec.len() != 2 {
+            return None;
+        }
+        Some(vec[1].clone())
+    }
+
+    fn reset_cb(success: bool, ctx: *mut c_void) {
+        if success {
+            info!(
+                "nvmf_event_handler: controller successfully reset {:?}",
+                success
+            );
+        } else {
+            error!("nvmf_event_handler: failed to reset controller ");
+        }
+        done_cb(ctx, success);
+    }
+
+    pub async fn reset_one_child(dev_name: String, ctx: *mut c_void) {
+        let ctrlr = match NVME_CONTROLLERS.lookup_by_name(&dev_name) {
+            Some(ctrlr) => ctrlr,
+            None => {
+                debug!(
+                    "device name  {:?} not a valid NVMe controller",
```

Comment:
```suggestion
                    "device name  {dev_name} not a valid NVMe controller"
```
Please replicate these changes across the traces. Btw strings don't need to use the debug trait (`:?`).